rosetta-source/pom.xml

```diff
@@ -872,6 +894,30 @@
                             </resources>
                         </configuration>
                     </execution>
+                    <!-- CodeList resource lifecycle - Ensure only CodeList JSON files are copied to target/classes/ -->
+                    <execution>
+                        <id>copy-json-codelist-resources</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <resources>
+                                <resource>
+                                    <directory>${project.basedir}/src/main/resources/org/isda/codelist</directory>
+                                    <includes>
+                                        <include>json/*.json</include>
+                                    </includes>
+                                    <excludes>
+                                        <exclude>xml/**</exclude>
+                                        <exclude>*.xsl</exclude>
+                                    </excludes>
+                                    <filtering>false</filtering>
+                                </resource>
+                            </resources>
+                            <outputDirectory>${project.build.outputDirectory}/org/isda/codelist</outputDirectory>
+                        </configuration>
+                    </execution>
```

Comment:
See note below: https://github.com/finos/common-domain-model/pull/3819/files#r2390598412

I don't think you need this.